sklearn/feature_selection/tests/test_sequential.py

```diff
@@ -145,3 +223,8 @@ def test_pipeline_support():
     pipe = make_pipeline(StandardScaler(), sfs)
     pipe.fit(X, y)
     pipe.transform(X)
+
+
+def test_raise_deprecation_warning():
+    with pytest.warns(FutureWarning):
```

Comment:
It will be better to match a partial string to be sure that we raise the expected warning:

```suggestion
    warn_msg = "Leaving ...."
    with pytest.warns(FutureWarning, match=warn_msg):
```